src/main/java/net/imagej/ops/image/ImageNamespace.java

```diff
@@ -441,6 +444,27 @@
 		return result;
 	}
 
+	// -- coordinate equation --
+	@OpMethod(op = net.imagej.ops.image.equation.DefaultCoordinatesEquation.class)
+	public <T extends RealType<T>> IterableInterval<T> equation(
+		final IterableInterval<T> out, final UnaryFunctionOp<long[], Double> in)
+	{
+		@SuppressWarnings("unchecked")
+		final IterableInterval<T> result = (IterableInterval<T>) ops().run(
+			net.imagej.ops.image.equation.DefaultCoordinatesEquation.class, out, in);
```

Comment:
This should be `ops().run(net.imagej.ops.Ops.Image.Equation.class, out, in)`. That way if someone writes another specialized `CoordinatesEquation` op, it could also be matched.